.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,39 @@
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
```

Comment:
no CI runs on branches?

Reply:
What would be the point of that?

Reply:
do you require PRs to be up to date before merging?
if not, then a passing PR could fail after merging to master?

Reply:
We don't, and I hadn't thought of that possibility. That being said, it will not prevent the PR from being merged.